### PR TITLE
node@*: remove npm resource.

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
Use upstream's npm. Users can still update this if desired.

`nodeenv` is a better replacement if you need your old Nodes built exactly to your requirements.

### CODE REVIEW

File: Formula/node@4.rb [hunk 1]
```diff
@@ -13,111 +14,15 @@ class NodeAT4 < Formula
...
-    args = %W[--prefix=#{prefix} --without-npm]
-    args << "--debug" if build.with? "debug"
-    args << "--shared-openssl" if build.with? "openssl"
-
-    if build.with? "full-icu"
-      args << "--with-intl=full-icu"
-    else
-      args << "--with-intl=small-icu"
-    end
+    args = ["-prefix=#{prefix}", "--with-intl=system-icu"]
```
Comment (ilovezfs) [inline Formula/node@4.rb]: --prefix not -prefix

**Comment by DomT4** [comment]
Same sort of question re actually removing these as in https://github.com/Homebrew/homebrew-core/pull/24346#issuecomment-367025990. Doubly so here since we don't actually need these older formulae ourselves, IIRC.

**Comment by MikeMcQuaid** [comment]
@DomT4 These packages are still supported by upstream and used by large numbers of users.

**Review by ilovezfs — APPROVED** [review]

**Comment by ilovezfs** [comment]
@MikeMcQuaid it looks like node@6 still has the npm resource. See https://github.com/Homebrew/homebrew-core/issues/24610.

**Pull request closed without merging**